config/locales/en.yml

```diff
@@ -244,6 +243,8 @@ en:
           Get support using your account.
         how-to-setup-your-account: >-
           Get support setting up your account.
+        communicating-with-jobseekers: >-
+          How to send and receive messages using the Applicant Tracking System.
```

Comment:
✂️ ?
I think this defines the text for the subcategory "Communicating With Jobseekers" before its cards.

We do not need this here since `communicating-with-jobseekers` is no longer a subcategory.

As above, "how-to-setup-your-account" text generates:

<img width="1033" height="602" alt="Image" src="https://github.com/user-attachments/assets/f9bb5c96-f17f-4300-99cc-7d8624415fd2" />